docs/src/main/sphinx/admin/fault-tolerant-execution.rst

```diff
@@ -409,7 +435,49 @@ load:
 
 .. code-block:: properties
 
-    exchange.base-directories=s3n://exchange-spooling-bucket-1,s3n://exchange-spooling-bucket-2
+    exchange.base-directories=s3://exchange-spooling-bucket-1,s3://exchange-spooling-bucket-2
+
+Google Cloud Storage
+~~~~~~~~~~~~~~~~~~~~
+
+If you are using Google Cloud Storage (GCS), it is recommended to configure a bucket lifecycle
+rule in GCS to automatically expire abandoned objects in the event of a node crash.
```

Comment:
We should probably move this and the other two near-identical statements to above the AWS S3 section and only say it once for all three, that way we're not repeating the same thing.